### DISCUSSION TITLE
MacVim: stable 9.0.472 (bottled) crashes on M1

### DISCUSSION THREAD

**Discussion by fahasch** [Homebrew/discussions#3858] [Everyday usage]
Having installed the latest bottled version of MacVim, I experience a crash when executing `mvim --clean -c "set showtabline=2"`. The MacVim application quits unexpectedly. I do not observe a similar behavior on an Intel mac.

showtabline is a rather innocent command (just showing the tab line). So I am not sure what can lead to such a strange behavior.

**Comment by SMillerDev** [comment]
Please report this to the macvim developers, Homebrew only packages the software
